package.json

```diff
@@ -63,5 +62,6 @@
     ],
     "importOrderSeparation": true,
     "importOrderSortSpecifiers": true
-  }
+  },
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```

Comment:
Can we transition to PNPM? I can do that change

Reply:
As a Bun employee I would like to suggest that if we are going to swap out yarn, we should use Bun as the package manager 😅 Any thoughts?

Reply:
Sure! why not?

Reply:
Cool, will do that soon.